xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -37,36 +37,59 @@
 
 namespace Xapian {
 
-    /** Utility function that prepares the 'training-data.txt' file in the current working directory. This file is used to train a model which in turn will be used to
-     *  assign scores to the documents based of Learning-to-Rank model. The file is created in the standard format of Letor training file
+    /** Utility function that prepares the 'training-data.txt' file in the
+     *  current working directory.
```

Comment:
The training data filename is no longer hard-coded, but this needs a tweak to make sense now - something like:

"Utility function that prepares the training data.

The training data file is used to train a model [...]"